example/nuxt.config.js

```diff
@@ -15,6 +15,10 @@ export default {
       }
     }
   },
+  // Nuxt.js plugins
+  plugins: [
```

Comment:
I was thinking of creating an internal plugin to enable the debuging process without need any custom plugins.
We can have `debug` option inside module options and register a our debug plugin if it is `true`
What do you think about it?

Reply:
I do totally agree. I'll update my PR with this change included!